app/templates/components/widgets/forms/image-upload.hbs

```diff
@@ -8,9 +8,14 @@
         <div class="ui text loader">{{t 'Image Uploading'}}</div>
       </div>
     {{/if}}
-    <div class="image">
-      <img src="{{selectedImage}}" alt="Selected image">
-    </div>
+    {{input type='file'
```

Comment:
This should be an image-upload property, Rather a property common only to profile pictures.
As this widget is used at other places too.

Reply:
@kushthedude Should I create a new widget for the same?